### PR TITLE
changes on meta

### PR DESCRIPTION
**Pull request by marianorod**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,16 +8,37 @@
...
+      <link rel="icon" type="image/png" sizes="32x32" href="images/icons/favicon-32x32.png"/>
+      <link rel="icon" type="image/png" sizes="16x16" href="images/icons/favicon-16x16.png"/>
+      <link href=”images/icons/apple-touch-icon.png"  sizes="180x180" rel="apple-touch-icon"/>
+      <link rel="mask-icon" href="images/icons/safari-pinned-tab.svg" color="#00DAB0"/>
+      <meta name="msapplication-TileColor" content="#00DAB0"/>
+      <meta name="theme-color" content="#00DAB0"/>
+      <link rel="manifest" href="manifest.json">
       <link href="https://fonts.googleapis.com/css?family=Titillium+Web" rel="stylesheet">
-      <meta name="msapplication-TileColor" content="favicon.ico">
       <meta name="title" content="Joystream: A user operated media platform">
```
Comment (bwhm) [inline index.html]: "A user governed video platform"

Comment (bwhm) [inline index.html]: "A user governed video platform"

**Review by bwhm — COMMENTED** [review]
Change the tagline as shown throughout.

File: index.html (hunk 1 above)
Comment (marianorod) [reply index.html]: Corrected

**Review by bwhm — APPROVED** [review]

**Pull request merged**